api/app/config/environment.js

```diff
@@ -0,0 +1,24 @@
+const DEVELOPMENT_ENV = 'development';
```

Comment:
You're probably going to want to get docker and manage.sh to work with this 

https://github.com/bcgov/court-of-appeal/blob/b631797066ae05fd395c85f83527b79533a1a5e1/docker/manage#L227

So it's straight forward for us to put the environment variables in Openshift.